src/Infrastructure.EntityFramework/Configurations/ApiKeyEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Bit.Infrastructure.EntityFramework.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Bit.Infrastructure.EntityFramework.Configurations;
+
+public class ApiKeyEntityTypeConfiguration : IEntityTypeConfiguration<ApiKey>
+{
+    public void Configure(EntityTypeBuilder<ApiKey> builder)
+    {
+        builder
+            .Property(s => s.Id)
+            .ValueGeneratedNever();
+
+        builder
+            .HasKey(s => s.Id)
+            .IsClustered();
+
+        builder
+            .HasIndex(s => s.UserId)
+            .IsClustered(false);
+
+        builder
+            .HasIndex(s => s.OrganizationId)
+            .IsClustered(false);
+
+        builder
+            .HasIndex(s => s.ServiceAccountId)
+            .IsClustered(false);
```

Comment:
No foreign key?

Reply:
It has a foreign key. The configuration is just used to setup indexes.